02/app.js

```diff
@@ -23,6 +23,33 @@ function setRandomPosition(element, error = null) {
 
 function initEventWithError(element, eventName, error) {
     element.addEventListener(eventName, function() {
-        setRandomPosition(this, error);
+       try {
+           setRandomPosition(this, error);
+        }
+       catch(e) {
+           setErrorinErrorPanel(e);
+        }
+        finally {
+            closeErrorPanel();
```

Comment:
Czy faktycznie tutaj musimy to wstawić? Jeśli nie będzie błędu to i nie będzie co zamykać ;)